docker/academy-chain-compose.yml

```diff
@@ -15,19 +15,15 @@ services:
       - NAME=Node01
       - NODE_KEY_FILE=/data/$NODE01_ACCOUNT_ID/p2p_secret.txt
       - PORT=30333
-      - RPC_PORT=9933
+      - RPC_PORT=9944
       - RUST_LOG=info
       - BOOT_NODES=/ip4/127.0.0.1/tcp/30333/p2p/$NODE01_PEER_ID
       - PUBLIC_ADDR=/ip4/127.0.0.1/tcp/30333
     volumes:
       - $BASE_PATH/:/data/
     expose:
-      - 30333
-      - 9933
       - 9944
     ports:
-      - 0.0.0.0:30333:30333
```

Comment:
that's a bit too much - if you remove the p2p port mapping nodes might not be able to talk